RELEASE_NOTES.md

```diff
@@ -129,7 +129,103 @@ include the new sigmoid module.  Options are now as follows:
+    element in a group.
+  
+  - The sort order of the shapes in mask manager groups has been
+    reversed so that the lowest ranking shape is at the bottom of the
+    group. The sort order of shapes outside of a group has also been
+    changed for consistency.
+  
+  - It is now possible for all mask types to be added continuously in
+    the mask manager.
+
+  - Fixed an issue where the brush was not properly displayed after being
+    created from the mask manager, and a crash when creating gradients from the
+    mask manager has also been fixed.
+
+  - For all shapes the editable state is now properly set after creation
+     making it possible to move and resize the different parts of the shapes.
+  
+  - Fixed a display issue when editing a shape name in the mask manager.
+  
+  - Fixed the state of the set operators in the mask manager -- when
+    moving a mask up/down we now ensure that the first mask has no
+    operator and that the second one always has an operator assigned.
+    If no operator has been set yet the default (union) operator is
+    used.
+  
+  - When using a shortcut to add shapes to a drawn (blending) mask the
+    blending mode will automatically switch to "drawn mask" or "drawn &
+    parametric mask", depending on what it was before, so that any newly
+    created shape will actually affect the image.
+  
+  - In the drawn mask blending mode there used to be an "invert mask"
+    option that had the same functionality as the "toggle polarity" option.
+    Since both were doing the same thing, "invert mask" is now removed.
+  
+  - The drawing of masks has been reworked to ensure that all types of
+    masks are drawn in the same way -- the central area, border, and
+    highlighted segments are now displayed consistently. The highlighted
+    segment is now more visible, especially for the brush mask, where
+    the highlighted segment was barely distinguishable due to a bug.
+
+  - In the style and copy/paste dialogs a new column has been added to
+    show whether a module uses a mask (drawn/parametric/raster) using
+    the standard mask icon.
+  
+  - Editing of drawn masks has been improved -- it is now easier to
+    select masks' control points and path segments (in some cases it was
+    easy to accidentally select the whole path mask rather than a single
+    segment). On-canvas mask rendering has also been improved for better
+    consistency between shape types.
+
+  - The brush path is now slightly more transparent in order to better
+    see the underlying image.
+  
+
+## Performance Improvements
+
+One of the major highlights of this release is the number of performance
+improvements and enhancements:
+
+- The following processing modules have had code cleanup and performance improvements.
   All SSE2 code paths have been removed (the optimized parallel code
   generated by the compiler is faster) or code optimized in the
   following modules. This has led to speed gains of 5-25%:
```

Comment:
"following modules" is now a repetition, so how about
  ... or code optimized, leading to speed gains of 5-40%.

(several modules have 40+% reductions in processing time)

Reply:
should be resolved now